packages/block-library/src/navigation/edit/navigation-menu-delete-control.js

```diff
@@ -11,11 +11,15 @@ import { useDispatch } from '@wordpress/data';
 import { useState } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 
-export default function NavigationMenuDeleteControl( { onDelete } ) {
+export default function NavigationMenuDeleteControl( {
+	onDelete,
+	kind,
+	type,
```

Comment:
All usages of these props within this file can be reverted to simply hardcoding them.